### PR TITLE
Add shuttle danger warning

### PR DESCRIPTION
**Pull request by itsmeow**
## About The Pull Request

If you are not attentive it's a bit difficult to tell what shuttle is dangerous or not, so foolish captains often buy dangerous shuttles.

This adds a warning level to shuttles in the communications console.

## Why It's Good For The Game

Makes it more clear what shuttle is dangerous, for the average comdon or traitor to enjoy.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

image

</details>

## Changelog
:cl:
add: Added dangerous shuttle warnings to the communications console.
tweak: Tweaked the description on Oh Hi Daniel shuttle to not reference "space roleplay".
tweak: Tweaked the description on STV5 shuttle to include the fact that the crates included are potentially explosive.
/:cl:

### CODE REVIEW

**Comment by mystery3525** [comment]
What if it isn't possible for regular captains to order dangerous shuttles in the first place

**Comment by itsmeow** [comment]
The only "high danger" shuttle they can order is The Arena, and that's only if bubblegum has been killed. Otherwise it's just subpar shuttles and the occasional Russian Bear shuttle. Everything else is locked behind emag.

**Comment by Fronsis** [comment]
Thank god, this is a needed change, i've experienced three rounds recently where Captains ordered it just because ''they didn't know they were like that'' or ''they wanted to see how it looked'' this will prevent them from doing so, they can just ask the radio.


A little relevant but the other day i had a discussion on discord with an antag that bought it 4noraisin because they wanted to see how it looked without proper objectives and upon talking with Admins they reminded me that i wasn't crazy that those shuttles are only allowed to be buyed only for antagonists with Hijack/Die a glorious death objective however when the PR that made those shuttles only visible/unlocked when using an emag got merged the section of the wiki of ''Inherently dangerous shuttles'' and the articles that said only hijack/DAGD can purchase them got removed, maybe i should bring this to the wiki channel..

**Comment by itsmeow** [comment]
Reminder: anyone can edit the wiki
and no one in the wiki channel will do it for you

**Comment by Fronsis** [comment]
Of course, this however was on a ruling section of the wiki, or part of the antagonist conduct i honestly don't remember but i'll need to check with an admin to double confirm that is the current policy regarding buying these shuttles.

**Comment by EvilDragonfiend** [comment]
this might need headmins' confirmation for what's subpar and danger, and if subpar counts to antagonism too.

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -299,20 +304,23 @@
...
 	admin_notes = "Flaming hot. The main area has a dance machine as well as plasma floor tiles that will be ignited by players every single time."
 	credit_cost = 10000
+	danger_level = SHUTTLE_DANGER_HIGH
 
 /datum/map_template/shuttle/emergency/arena
 	suffix = "arena"
 	name = "The Arena"
 	description = "The crew must pass through an otherworldy arena to board this shuttle. Expect massive casualties. The source of the Bloody Signal must be tracked down and eliminated to unlock this shuttle."
-	admin_notes = "RIP AND TEAR."
+	admin_notes = "RIP AND TEAR. Creates an entire internal Z-level where you have to kill each other in a massive battle royale to get to the actual shuttle."
```
Comment (Archanial) [inline code/datums/shuttles.dm:321]: please replace "battle royale"

**Review by Archanial — DISMISSED** [review]

File: code/datums/shuttles.dm (hunk 1 above)
Comment (Archanial) [reply code/datums/shuttles.dm:321]: Don't confuse react me it's not battle royale and I'm sick of hearing this @Tyranicranger4

Comment (itsmeow) [reply code/datums/shuttles.dm:321]: Then what IS it

Comment (Archanial) [reply code/datums/shuttles.dm:321]: Certainly not battle royale
Especially since we have actual battle royale gamemode

Comment (itsmeow) [reply code/datums/shuttles.dm:321]: I've seen this shuttle ingame twice. The one time I remember I got murdered in a massive chainsaw fight to the death. How is that not a battle royale

Comment (Penwin0) [reply code/datums/shuttles.dm:321]: Just change it to a synonim like deathmatch or Free for all

Comment (itsmeow) [reply code/datums/shuttles.dm:321]: it completely matches the definition of a battle royale. also no one sees the description except admins. It very accurately describes what occurs (a battle royale)

Comment (Penwin0) [reply code/datums/shuttles.dm:321]: I don't think it matters much either, it's an admin only thing anyhow

**Review by Penwin0 — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**